components/layout/tabBar/TabBar.native.tsx

```
@@ -0,0 +1,119 @@
+      accessibilityRole={Platform.OS === "web" ? "link" : "button"}
+      accessibilityState={isFocused ? { selected: true } : {}}
+      accessibilityLabel={options.tabBarAccessibilityLabel}
+      onPress={onPress}
+      onLongPress={onLongPress}
+    >
+      {options.tabBarIcon &&
+        options.tabBarIcon({
+          focused: isFocused,
+          color: isFocused ? "white" : SECONDARY_COLOR,
+          size: 0,
+        })}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)